packages/cli/src/commands/redis/promote.ts

```diff
@@ -11,7 +11,7 @@ export default class Promote extends Command {
   }
 
   static args = {
-    database: Args.string({required: false}),
+    database: Args.string({required: false, description: 'Name of the Redis database. If omitted, it will default to the primary Redis instance associated with the app.'}),
```

Comment:
Change to "name of the Key-Value Store database. If omitted, it defaults to the primary instance associated with the app."